wallet/src/account/mod.rs

```diff
@@ -146,8 +146,12 @@ impl Account {
         let network_fee = Amount::from_atoms(10000);
 
         let utxos_by_currency = group_utxos_for_input(
-            self.get_utxos(UtxoType::Transfer | UtxoType::LockThenTransfer, median_time)
-                .into_iter(),
+            self.get_utxos(
+                UtxoType::Transfer | UtxoType::LockThenTransfer,
+                median_time,
+                true,
```

Comment:
This should be controllable by the user at the wallet controller level.

Reply:
Added utxo_states as argument for get_balance and get_utxos